OpenEdXMobile/src/main/java/org/edx/mobile/model/parser/AuthResponseMapper.kt

```diff
@@ -0,0 +1,24 @@
+package org.edx.mobile.model.parser
+
+import org.edx.mobile.model.authentication.AuthResponse
+import org.edx.mobile.util.TimeUtils
+
+/**
+ * Auth response mapper to extend the functionality of [AuthResponse] model. This class takes
+ * responsibility to add or modify all members, so any change in server response should reflect here
+ * to keep it consistent.
+ */
+class AuthResponseMapper : ResponseMapper<AuthResponse, AuthResponse> {
+
+    override fun map(response: AuthResponse): AuthResponse {
```

Comment:
user `Deserializer` approach instead of mapper, cuz in the current situation `ResponseMapper` used the same type.